### ISSUE TITLE
Admonitions don't render after the update to 4.2.0

### ISSUE THREAD

**Issue by savyajha** [valentine195/obsidian-admonition#34]
My admonitions don't render after the update to 4.2.0. I get the following error in the console:

```
TypeError: Cannot read property 'trim' of undefined
    at ObsidianAdmonition.eval (eval at <anonymous> (app.js:1), <anonymous>:9921:35)
    at Generator.next (<anonymous>)
    at eval (eval at <anonymous> (app.js:1), <anonymous>:31:71)
    at new Promise (<anonymous>)
    at __awaiter (eval at <anonymous> (app.js:1), <anonymous>:27:12)
    at ObsidianAdmonition.postprocessor (eval at <anonymous> (app.js:1), <anonymous>:9899:16)
    at app.js:1
    at Function.t.postProcess (app.js:1)
    at t.postProcess (app.js:1)
    at h (app.js:1)
```

Attaching a screenshot for reference:

<img width="1440" alt="Screenshot 2021-05-20 at 9 17 34 PM" src="https://user-images.githubusercontent.com/3996019/119009534-d8589a00-b9b0-11eb-9b75-6505f70f93b9.png">

**Comment by valentine195** [comment]
Sorry about that, should be fixed in 4.2.1

**Comment by savyajha** [comment]
No issues. By golly this is the fastest I've ever seen a bug get closed!
